### DISCUSSION TITLE
How do I receive RTSP video streams from VLC media player

### DISCUSSION THREAD

**Discussion by Violencekc** [PlusToolkit/PlusLib#867] [Q&A]
I successfully used RTSP stream in VLC Media Player, but Connection failed in plus Server launch! Here is my config,Please give me some help. Thank you very mu
[PlusDeviceSet_Server_OpenCVVideoSource.txt](https://github.com/PlusToolkit/PlusLib/files/7647234/PlusDeviceSet_Server_OpenCVVideoSource.txt)
ch

**Comment by adamrankin** [comment]
Please provide the error log

**Reply by Violencekc** [reply]
[the error log.txt](https://github.com/PlusToolkit/PlusLib/files/7657611/the.error.log.txt)
